ocs_ci/templates/ocs-deployment/multicluster/odr_s3_secret.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+data:
+  AWS_ACCESS_KEY_ID: PLACE_HOLDER 
+  AWS_SECRET_ACCESS_KEY: PLACE_HOLDER 
+kind: Secret
+metadata:
+  # Replace PLACE_HOLDER_FOR_REGION with region name(east, west) etc
+  name: odr-s3secret-PLACE_HOLDER_FOR_REGION
+  namespace: odr-system
```

Comment:
`odr-system` to be replaced by `openshift-dr-system`

Reply:
done